src/main/webapp/index.jsp

```diff
@@ -15,5 +15,6 @@ limitations under the License.
 --%>
 <jsp:include page="/vendor/startbootstrap/template.jsp">
   <jsp:param name="content" value="index-content"/>
+  <jsp:param name="scriptFile" value="indexPage"/>
```

Comment:
just a quick comment before full review - what do you think about `pages/index.js` to organize all the page files?

Reply:
I like it.

Reply:
Should there also be a module folder?

Reply:
Yea, maybe `lib/`?